build.py

```diff
@@ -58,6 +58,8 @@ def docker_build(image_spec, path, build_args):
     for k, v in build_args.items():
         command += ['--build-arg', f'{k}={v}']
     command.append(path)
+    # debug
+    print(command)
     subprocess.check_call(command, shell=True)
```

Comment:
@yuvipanda - I'm not getting any output from this call and the onbuild image isn't showing up. Any thoughts on what is going on here?

Reply:
Any output here? https://github.com/pangeo-data/pangeo-stacks/pull/114/files#diff-2955d5257f635de1df53f55a171ca5c7R139